### PR TITLE
Show target magnitudes from DB and catalog.

### PR DESCRIPTION
**Pull request by rpiaggio**
It seems we can't send magnitudes to the API yet, so it's flickering when querying the catalog (the new magnitudes come from the catalog, are shown in the UI, but a second later they are refreshed from the DB):

Kapture 2020-11-04 at 17 07 08

Therefore I'm opening the PR as draft since I don't know if we want to include this in the milestone. Or we could tag it with "WORK IN PROGRESS".

### CODE REVIEW

File: common/src/main/scala/explore/model/reusability.scala [hunk 1]
```diff
@@ -25,6 +27,8 @@ object reusability {
   // TODO: Move to lucuma-ui.
   implicit def quantityReuse[N: Reusability, U]: Reusability[Quantity[N, U]] =
     Reusability.by(_.value)
+  implicit val magnitudeValueReuse: Reusability[MagnitudeValue]              = Reusability.byEq
```
Comment (cquiroz) [inline common/src/main/scala/explore/model/reusability.scala:30]: I thought we had these on lucuma-ui. If not we could move them there later on

**Review by cquiroz — APPROVED** [review]
It would be nice to include it but I agree that the revert to db values seems odd.
I wouldn't mind including it if we can fix that

File: common/src/main/scala/explore/model/reusability.scala (hunk 1 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/model/reusability.scala]: I now moved them to lucuma-ui.

**Comment by rpiaggio** [comment]
I hid the magnitudes editor for now and included the fix for when tracking info from the catalog is missing. Also, marked the proposal editor as WIP.

**Comment by swalker2m** [comment]
Sorry guys, I dropped the ball on this by not including magnitudes in the editor input.  I just need to figure out how to indicate when magnitude values should be deleted.  I was planning on rolling it with the unset feature in general.

**Comment by rpiaggio** [comment]
> Sorry guys, I dropped the ball on this by not including magnitudes in the editor input. I just need to figure out how to indicate when magnitude values should be deleted. I was planning on rolling it with the unset feature in general.

No worries.

I'm not sure that "resetting" a list is the same as "unsetting" it but I will continue the discussion in https://github.com/gemini-hlsw/lucuma-tmp-api/pull/62.

**Pull request merged**